src/awsexplorer/commands/copyArn.ts

```diff
@@ -43,9 +41,5 @@ export async function copyArnCommand(
                     showLogOutputChannel()
                 }
             })
-        recordCopyArn({ result: 'Failed' })
     }
 }
-
-// TODO add telemetry for copy arn
-function recordCopyArn({ result }: { result: telemetry.Result }): void {}
```

Comment:
:/

Reply:
haha. we get this with vscode_executeCommand at least now? But we should do a proper one.

Reply:
Yup it's covered by commands now. Need to remove all those try/catches though to get the result/reason.